library/src/main/java/com/wultra/android/powerauth/networking/Api.kt

```diff
@@ -135,12 +135,16 @@ abstract class Api(
         listener: IApiCallResponseListener<TResponseData>) {
 
         var bytes = bodyBytes
+        val headers = headers ?: HashMap<String, String>()
```

Comment:
Or the empty HashMap can be the default value for the argument which can be non-nullable.